modules/nf-core/pilon/meta.yml

```
@@ -0,0 +1,71 @@
+name: "pilon"
+description: Automatically improve draft assemblies and find variation among strains, including large event detection
+keywords:
+  - polishing
+  - assembly
+  - variant calling
+tools:
+  - "pilon":
+      description: "Pilon is an automated genome assembly improvement and variant detection tool."
+      homepage: "https://github.com/broadinstitute/pilon/wiki"
+      documentation: "https://github.com/broadinstitute/pilon/wiki/Requirements-&-Usage"
+      tool_dev_url: "https://github.com/broadinstitute/pilon"
+      doi: "https://doi.org/10.1371/journal.pone.0112963"
+      licence: "['GPL v2']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - fasta:
+      type: file
+      description: FASTA file
+      pattern: "*.{fasta}"
```

Sorry, I have always just a few minutes, I know that isnt ideal. But looking at the pilon documentation I found that this seem to be the assembly, so it might be good to detail that here:
```suggestion
  - fasta:
      type: file
      description: FASTA of the input genome
      pattern: "*.{fasta}"
```